sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+# sonarqube server url
+sonar.host.url=https://sonarqube.cf.ai
```

Comment:
@robertwenquan please review to make sure there are no secrets in this PR please. This is merging all private repo into public for good.

Reply:
there is no secrets in this file. But this file might not be good for public consumption. 
The `sonarqube.cf.ai` itself should be internal only